client/src/game/ui/tools/select.vue

```diff
@@ -111,7 +109,11 @@ export default class SelectTool extends Tool {
             this.selectionHelper.w = 0;
             this.selectionHelper.h = 0;
 
-            layer.selection = [this.selectionHelper];
+            if (keyboardState.shiftPressed || keyboardState.ctrlPressed) {
+                // If either control or shift are pressed, do not remove selection
```

Comment:
In this case and the above, the selection helper should respectively be increased or added to have consistent behaviour.

Reply:
I do not understaund what you mean.

Reply:
Before starting the new selection with shift, a selection box is already present, after dragging the new box, the old one is removed but there are still elements from that selectionbox selected, but possibly outside of the new box.

This feels kinda strange, so I would suggest to make the original selectionbox bigger to fit all previous as well as all new shapes.  Or we just get rid of the selectionbox all together as you alluded to earlier.

Reply:
I do not like making the previous selectionBox bigger, as that would cover shapes that the user does not want to select, and that would be confusing. Just removing the selectionbox would be cleaner IMO.

Reply:
I can live with that, but that's for another PR, so this is OK.